_markbind/layouts/default/navigation.md

```diff
@@ -1,6 +1,6 @@
 <navbar placement="top" type="inverse">
   <a slot="brand" href="/" title="Home" class="navbar-brand">Software Engineering for Self-Directed Learners</a>
   <li slot="right">
-    <input id="searchbar" placeholder="Search" class="form-control"> 
+    <searchbar placeholder="Search" algolia menu-align-right>

```

Comment:
Shouldn't there be a `/>`?

Reply:
Fixed 🙂 

Preview was generated using an older version of MarkBind. Latest preview should be working now!

Reply:
> Fixed

Thanks. I noticed the same error in the MarkBind user guide.

Reply:
Made a PR to fix that as well. Thanks for catching it!